client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -82,6 +83,8 @@ export const useUrlQueryParams = ({
     [updateQuery]
   );
 
+  const clearSort = () => updateQuery({ sort: undefined, dir: undefined });
```

Comment:
We didn't currently have a way to clear sorting - also had to change the above useEffect to only reset the initial sort on page mount, rather than any time we don't have a currently applied sort.